R/transform-calculate.R

```diff
@@ -86,23 +86,37 @@ collateCats <- function (inserts, var_cats) {
     # setup an empty AbstractCategories object to collate into
     cats_out <- AbstractCategories()
     cats_out@.Data <- var_cats
+    
+    if (length(var_cats) < 1) {
+        halt("Can't collateCats with less than one categories.")
```

Comment:
I'm not a linguist, but "one categories" looks wrong to me. And "less" should be "fewer". But aren't you really trying to say "no categories"?